### DISCUSSION TITLE
Can I organize resources into subfolders?

### DISCUSSION THREAD

**Discussion by uadoronin** [filamentphp/filament#3215] [Help] [answered]
Hello friends 🙂!

I have many resources in the _**Filament/Resources/**_ folder. 
Is there any way to organize the structure by putting resources in subfolders?

When I create a resource like this:
`php artisan filament:resource System/Admin`

I get a router name that doesn't work:
`GET|HEAD 
local.site/system/admins 
filament.resources.system/admins.index`

Thank you in advance for your reply.

**Comment by danharrin** [comment]
Hey! Why doesn't that router name work?

**Comment by uadoronin** [comment, marked as answer]
Sorry, my mistake. After generating the resource, the path to the model was wrong.
Everything works.
Thank you for the Filament!
